src/main/resources/db/mysql/petclinic_db_setup_mysql.txt

```diff
@@ -23,7 +23,15 @@
 
 3) Run the app with `spring.profiles.active=mysql` (e.g. as a System property via the command
    line, but any way that sets that property in a Spring Boot app should work).
+   
+4) For newer versions of Spring Boot, those will fork the JVM that executes the webserver,
+   which means properties on the command line are lost. You can use:
+    
+   mvn spring-boot:run\
+   -Dspring-boot.run.jvmArguments="-Dspring.profiles.active=mysql -Dspring.datasource.initialization-mode=always"
+    
+   To set profiles & properties.
 
 N.B. the "petclinic" database has to exist for the app to work with the JDBC URL value
 as it is configured by default. This condition is taken care of by the docker-compose
-configuration provided, or by the `schema.sql` if you can run that as root.
\ No newline at end of file
+configuration provided, or by the `schema.sql` if you can run that as root.
```

Comment:
Looks like an unrelated change, please revert.

Reply:
Github's web editor added a newline at the end of the file automatically. Fixed.